pico/api/src/main/java/io/helidon/pico/DefaultQualifierAndValue.java

```diff
@@ -17,12 +17,15 @@
 package io.helidon.pico;
 
 import java.lang.annotation.Annotation;
+import java.util.Map;
 import java.util.Objects;
 
-import io.helidon.pico.types.AnnotationAndValue;
-import io.helidon.pico.types.DefaultAnnotationAndValue;
-import io.helidon.pico.types.DefaultTypeName;
-import io.helidon.pico.types.TypeName;
+import io.helidon.common.types.AnnotationAndValue;
+import io.helidon.common.types.DefaultAnnotationAndValue;
+import io.helidon.common.types.DefaultTypeName;
+import io.helidon.common.types.TypeName;
+
+import jakarta.inject.Named;
```

Comment:
Constants should not exist on an implementation (such as`NAMED` and `WILDCARD_NAMED`). There seems to be a cyclic dependency for Wildcard_named, but this should be done some other way (as the current approach is inconsistent with other `Default*` which are generated)